pkg/proxy/ipvs/proxier.go

```diff
@@ -274,11 +274,11 @@ func (r *realIPGetter) NodeIPs() (ips []net.IP, err error) {
 // Proxier implements ProxyProvider
 var _ proxy.ProxyProvider = &Proxier{}
 
-// ParseExcludedCIDRs parses the input strings and returns net.IPNet
+// parseExcludedCIDRs parses the input strings and returns net.IPNet
 // The validation has been done earlier so the error condition will never happen under normal conditions
-func ParseExcludedCIDRs(excludeCIDRStrs []string) []*net.IPNet {
+func parseExcludedCIDRs(excludeCIDRs []string) []*net.IPNet {
 	var cidrExclusions []*net.IPNet
-	for _, excludedCIDR := range excludeCIDRStrs {
+	for _, excludedCIDR := range excludeCIDRs {
 		_, n, err := net.ParseCIDR(excludedCIDR)
 		if err == nil {
```

Comment:
Do you think it still makes sense to flip the err check here and add logging if ParseCIDR fails?
```go
if err != nil {
     klog.Errorf("Error parsing exclude CIDR %q,  err: %v", excludedCIDR, err)
     continue
}

cidrExclusions = append(cidrExclusions, n)
```

Reply:
I have changed the code as suggested.